hack/generate-manifest-flow-visibility.sh

```diff
@@ -23,8 +23,17 @@ function echoerr {
 _usage="Usage: $0 [--mode (dev|release)] [--keep] [--help|-h]
 Generate a YAML manifest for the Clickhouse-Grafana Flow-visibility Solution, using Kustomize, and
 print it to stdout.
-        --mode (dev|release)  Choose the configuration variant that you need (default is 'dev')
-        --keep                Debug flag which will preserve the generated kustomization.yml
+        --mode (dev|release)        Choose the configuration variant that you need (default is 'dev').
+        --keep                      Debug flag which will preserve the generated kustomization.yml.
+        --volume (ram|pv)           Choose the volume provider that you need (default is 'ram').
+        --storageclass -sc <name>   Provide the StorageClass used to dynamically provision the 
+                                    Persistent Volume for ClickHouse storage.
+        --local <path>              Create the Persistent Volume for ClickHouse with a provided
+                                    local path.
+        --nfs <hostname:path>       Create the Persistent Volume for ClickHouse with a provided
+                                    NFS server hostname or IP address and the path exported in the
+                                    form of hostname:path.
+        --no-ch-monitor             Generate a manifest without the ClickHouse monitor.
```

Comment:
I feel like instead of adding all these flags we should work on adding helm support for the flow visibility solution right away (see https://github.com/antrea-io/antrea/pull/3578) as it provides a more natural / standardized way of generating the manifests than our custom `generate-manifest` scripts. Configuring a PV / PVC would be much easier with helm charts.

Reply:
It sounds good to add helm support for the flow visibility solution. But as we are migrating from the main repo to a child repo [theia](https://github.com/antrea-io/theia), the purpose of opening this PR in main repo is to include PV support for the first round of CQE testing.  I wonder if it is possible to have the PV support with Kustomize first. When we finish the migration, we can add helm support on the child repo side.

Reply:
I think we should definetely have a target to provide a helm chart for flow visibility in 1.7, just like Antonin is doing for Antrea.

I also think this can be done in a dedicated PR, to avoid conflating too many changes into a single PR.